software/jtr/buildroot-external/package/john/john.mk

```diff
@@ -0,0 +1,38 @@
+################################################################################
+#
+# John the Ripper
+#
+################################################################################
+
+JOHN_VERSION = riscv
```

Comment:
Alternatively, an explicit commit hash could be specified here instead of fetching the head of the branch, which might be safer.